modules/cluster-logging-log-forwarding-disable.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-log-forwarding.adoc
+
+[id="cluster-logging-log-forwarding-disable_{context}"]
+= Disabling the Log Forwarding feature
+
+To disable the Log Forwarding feature, remove the `clusterlogging.openshift.io/logforwardingtechpreview:enabled` parameter from the Cluster Logging CR and delete the Log Forwarding CR. The container and node logs will be forwarded to the internal {product-title} Elasticsearch instance.
```

Comment:
I'd define custom resource (CR) here, and then not define it in the _Procedure_ that follows later.